test/algorithms/set_operations/union/union_multi.cpp

```diff
@@ -393,18 +388,10 @@ void test_areal()
 
     {
         ut_settings settings;
-#if !defined(BOOST_GEOMETRY_NO_ROBUSTNESS) && !defined(BOOST_GEOMETRY_TEST_INCLUDE_FAILING_TESTS)
-        settings.test_validity = false;
-#endif
-
+        settings.test_validity = BG_IF_RESCALED(false, true);
         test_one<Polygon, MultiPolygon, MultiPolygon>("ticket_9081",
             ticket_9081[0], ticket_9081[1],
-#if defined(BOOST_GEOMETRY_NO_ROBUSTNESS)
-            3,
-#else
-            4,
-#endif
-            0, 31, 0.2187385,
+            BG_IF_RESCALED(4, 3), 0, 31, 0.2187385,
```

Comment:
So one of these numbers is not correct, right?

Reply:
I will merge Wednesday and check this. It might be both right (rescaling might split a polygon which would otherwise be unsplit). The reverse case we also had (should actually be split but is merged in rescaling).
Thank you.

Reply:
No, I mean in general, in all of these cases where the result depends on rescaling. I'm just curious if you consider both results correct or not. Previously some of the tests were enabled only if the `FAILING_TESTS` flag was enabled, but now this macro is used. So it's not clear which expected result is the correct one.

Reply:
But also in general, as far as I'm aware and can think of now, all results are correct (unless marked as _IGNORE when invalid). Otherwise they're excluded from the suite. Or a comment explicitly notes the reason of failure.
I did not remove FAILING_TESTS flags but renamed some of them, and (when denoting invalidity or skipping a testcase) that is handled in the test-header itself now.

Anyway, there are not so much of these new IF_RESCALED flags, I just checked most of them and added some comments. In general I consider the non-rescaled versions (often more splitted) as more correct. We've to check in the next phase what the final result will be.